cmd/kogito/command/install/setup_test.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package install
+
+import (
```

Comment:
can't this file be reused? seems identical from the previous one.

Reply:
See:
https://golang.org/pkg/testing/#hdr-Main

We need to have a test setup in each package. Unfortunately all of then is the same, we can't reuse it.

Reply:
Actually, I was able to reuse, but if the `TestMain` would need to have more features, this should be reviewed.

Take a look and come up with your conclusions.

Reply:
It s trick to "reuse" it since it's the way golang works.

Reply:
See the "reusable" function: 

```go
func SetupTestsWithMockKubeContext(t *testing.M) (code int) {
	teardown := OverrideKubeConfigAndCreateDefaultContext()
	code = t.Run()
	teardown()
	return code
}
```
See the use:
```go
func TestMain(t *testing.M) {
	os.Exit(test.SetupTestsWithMockKubeContext(t))
}
```

Did you understand why we won't be able to reuse it?